lib/loaders/total.js

```diff
@@ -5,7 +5,7 @@ import httpLoader from './http';
 
 export const total = ({ path, options }) => {
   const key = toKey(path, assign({}, options, {
-    size: 1,
+    size: 0,
```

Comment:
This change depends on https://github.com/artsy/gravity/pull/9992 being deployed.